client/my-sites/site-settings/comment-display-settings/index.jsx

```diff
@@ -0,0 +1,48 @@
+/**
+* External dependencies
+*/
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import FormSelect from 'components/forms/form-select';
+import FormTextInput from 'components/forms/form-text-input';
+
+const CommentDisplaySettings = ( { translate, onChangeField, submittingForm, fields } ) => {
+	return (
+		<FormFieldset className="comment-display-settings">
+			<div className="comment-display-settings__setting">
+				<FormLabel htmlFor="highlander_comment_form_prompt">{ translate( 'Comments Label' ) }</FormLabel>
```

Comment:
Alternate string suggestion: 
* `translate( 'Comment and Featured Labels' )`&mdash; translations: **25**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/settings-discussion-comment-label)